From: Christian Kellner <christian@kellner.me>
Date: Thu, 19 Apr 2018 14:22:04 +0200
Subject: thunderbolt: sync D-Bus API with bolt changes

bolt 0.3 deprecated all AUTHORIZED_XXX status flags, but added
a CONNECTING one. Also AuthFlags got renamed to AuthCtrl.
---
 js/ui/status/thunderbolt.js | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/js/ui/status/thunderbolt.js b/js/ui/status/thunderbolt.js
index 6c97ac8..85a8910 100644
--- a/js/ui/status/thunderbolt.js
+++ b/js/ui/status/thunderbolt.js
@@ -56,12 +56,11 @@ const BoltDeviceProxy = Gio.DBusProxy.makeProxyWrapper(BoltDeviceInterface);
 
 var Status = {
     DISCONNECTED: 'disconnected',
+    CONNECTING: 'connecting',
     CONNECTED: 'connected',
     AUTHORIZING: 'authorizing',
     AUTH_ERROR: 'auth-error',
-    AUTHORIZED: 'authorized',
-    AUTHORIZED_SECURE: 'authorized-secure',
-    AUTHORIZED_NEWKEY: 'authorized-newkey'
+    AUTHORIZED: 'authorized'
 };
 
 var Policy = {
@@ -70,7 +69,7 @@ var Policy = {
     AUTO: 'auto'
 };
 
-var AuthFlags = {
+var AuthCtrl = {
     NONE: 'none',
 };
 
@@ -141,7 +140,7 @@ var Client = new Lang.Class({
     },
 
     enrollDevice(id, policy, callback) {
-	this._proxy.EnrollDeviceRemote(id, policy, AuthFlags.NONE,
+	this._proxy.EnrollDeviceRemote(id, policy, AuthCtrl.NONE,
                                        (res, error) => {
 	    if (error) {
 		callback(null, error);
